src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -198,6 +199,25 @@ export class RoslynLanguageServer {
             );
         });
 
+        this._languageClient.onNotification(RoslynProtocol.ProjectLoadFailureNotification.type, async (failure) => {
```

Comment:
If we have a bunch of notifications at once are we going to spam the user? Or does VS Code throttle this in some way for us?

Reply:
+1. Can we kick off an array of projects and 1 collated exception that failed load instead?

Reply:
from testing it looks like only 1 shows at once

Reply:
@dibarbet Once that's dismissed though, does the next one appear? Or the other are just ignored?

Reply:
the others are ignored.  I was OK with that because these just point to the logs which will show all the errors.